packages/TelegramClient-Tests.package/TCTAcceptanceTests.class/instance/setUp.st

```diff
@@ -1,8 +1,5 @@
 running
 setUp
 
-	self core: TCTTestCore new.
-	10 seconds wait.
-	self core authHandler isAuthorizationStateReady ifFalse: [
-		self core loginWithTestData.
-		10 seconds wait].
\ No newline at end of file
+	self core: TCTNoTDLibCore new.
+	self core chats add: (TCCChat new id: TCTMocks mockChatID)
```

Comment:
I'm not entirely sure this will work. Once you merge the group popups (which are included in develop already), some of the tests in AcceptanceTests do require a somewhat working core (i.e. to open a working TelegramClient on it). 

Maybe you could use the NoTDLibCore only for your specific tests for the time being (until the test refactoring) next spring?

Reply:
There is actually a similar thing going on in feature/show_chat_member_count. There, the mock client also replaces the TCTTestCore that is used before. This issue only has to be resolved once, so maybe just figure it out together with the others

> It is a bit weird, that TCTUITest has test cases in the first place. Maybe we should rename to TCTUITestCase and only place the test cases in subclasses? We placed all of ours into TCTAcceptanceTests.
> 
> How about you put your method into the subclass? Your mock core should also already work for `testCanOpenChatInfoPage` and `testCanCloseChatInfoPage` as long as some group chats are available. I think making them use the subject you create instead of creating their own with a mock core would be great.
> 
> The only issue is having both those tests and the testMessageAuthors in the same class, when they need different things for setup. I think the user-store MR could solve that though, right @phkeese ? (Especially when the NoTDLibCore and the MockCore are merged into the same thing, then creating one of those in the setUp of TCTAcceptanceTests would be all we ever need).

_Originally posted by @rsommerfeld in https://github.com/hpi-swa-teaching/TelegramClient/pull/381#discussion_r642053292_

Reply:
we use a TCTNoTDLibCore specifically for this test, so this should not be a problem